aria_extension_tests/cloudify_aria_adapters/test_context_adapter.py

```diff
@@ -304,6 +303,13 @@ def mock_workflow(ctx, graph):
                     operation_name,
                     operation_kwargs=op_dict
                 )
+                if inputs:
```

Comment:
I don't follow you. They are passed only once, in line 317. Lines 306-312 are only about creating the input definitions in the operation.